src/app/introduction/introduction.component.scss

```diff
@@ -0,0 +1,145 @@
+@import "../styles/components/_variables";
+
+::selection {
+    background: $yellow; /* WebKit/Blink Browsers */
+}
+::-moz-selection {
+    background: $yellow; /* Gecko Browsers */
+}
+h3 {
+    text-align: center;
+    font-weight: $font-bold;
+    color: $blue;
+}
+h2 {
+    font-weight: $font-bold;
+}
+
+.mosquito {
+    width: 80px;
+    height: auto;
+}
+
+.lead-title {
+    font-weight: $font-bold;
+    font-size: 2.5em;
+    color: $black;
+}
+
+.lead-sub-heading {
+    color: grey;
+    font-weight: $font-bold;
+    font-size: 1.5em;
+}
+
+@mixin border-radius($radius1, $radius2) {
+    -webkit-border-bottom-right-radius: $radius1 $radius2;
+    -moz-border-radius-bottomright: $radius1 $radius1;
+    border-bottom-right-radius: $radius1 $radius1;
+}
+
+.rectangle-speech-border {
+  position: relative;
+  padding: 50px;
+  width: 100%;
+  margin: 1em 0 3em;
+  border: 2px solid black;
+  color: #333;
+  background: $white;
+  -webkit-border-radius: 50px;
+  -moz-border-radius: 50px;
+  border-radius: 50px;
+  &:before,
+  &:after {
+    content: "";
+    position: absolute;
+    z-index: 10;
+    bottom: -30px;
+    height: 30px;
+    border-style: solid;
+    border-width: 0 2px 2px 0;
+    border-color: $black;
+    background: transparent;
+    display: block;
+  }
+  &:before {
+    left: 50px;
+    width: 50px;
+    @include border-radius(80px, 50px);
+  }
+  &:after {
+    left: 55px;
+    width: 20px;
+    @include border-radius(40px, 50px);
+  }
+  > :first-child:after {
+    content: "";
+    position: absolute;
+    bottom: -10px;
+    left: 76px;
+    width: 24px;
+    height: 15px;
+    background: $white;
+  }
+}
+
+.arrow {
+    width: 75px;
+    line-height: 75px;
+    text-align: center;
+    opacity: 1;
+    border: none;
+    color: $white;
+    border-radius: 50%;
+}
+
+.arrow-red {
+    @extend .arrow;
+    background: #d9534f;
```

Comment:
1. capitalize the letters for consistency with https://github.com/systers/PC-Prep-Kit/pull/27/files#diff-63eda3fc63b2d7288a308dc99a98dba9
2. using variables?